### ISSUE TITLE
Mavenize and remove Eclipse project files?

### ISSUE THREAD

**Issue by mrice** [Kotlin/kotlin-koans#64]
Hey folks,

Just getting into Kotlin and I had a hard time importing the koans into Intellij. Can we make it a Maven project and/or remove the editor settings (e.g., Eclipse files like .project)?

**Comment by hhariri** [comment]
What was the exact issue on importing? It should basically just open up.

**Comment by mrice** [comment]
Hmmm... ok, maybe it's a documentation/getting started issue (or I just am not following the expected use case). When I try to import it with Intellij (ultimate) as existing sources, I had a bunch of issues. When I simply converted to a Maven-style project I was able to import cleanly. 

I know "a bunch of issues" doesn't help a bunch though. Let me retry and give you the specific errors.

**Comment by mrice** [comment]
Hmm... not sure I can say what my original issue was. But I still think it's valid to get rid of the IDE-specific files.

I just re-ran my first-time experience with Intellij Idea. I elected to import the project from existing sources. Using the wizard, I pass over the step where it asks for an external model. Then Idea won't let me proceed without overwriting the .idea file, which I wasn't sure what to think of. 

Same issue with an existing .iml file. I have to chose whether to overwrite or reuse, which I frankly don't know what to chose since I just elected to overwrite the .idea directory.  When I select "reuse" I get a weird blank dialog that says "no frameworks detected." Same if I pick "overwrite."

On this last run, it looks like Intellij picked up the Kotlin runtime stuff, which could have been left over from my last runs through this and now I'm able to run the tests for the koans. I wish I could tell you what the original issue was.

In any event, it seems to me like Maven is a universal structure for either the Eclipse or Idea audience that might be seeing Kotlin for the first time, and it would remove the noise and confusion introduced by either IDE from the github repo.

Thanks for taking a look though.
